modules/renter/hostdb/hostdb.go

```diff
@@ -115,3 +119,22 @@ func New(cs modules.ConsensusSet, wallet modules.Wallet, tpool modules.Transacti
 
 	return hdb, nil
 }
+
+// newHostDB creates a HostDB using the provided dependencies. Unlike New, it
+// does not have any side effects (i.e. it does not spawn background threads,
+// perform I/O, or call stateful methods of its dependencies.)
+func newHostDB(w hdbWallet, tp hdbTransactionPool, d hdbDialer, s hdbSleeper, p hdbPersister, l hdbLogger) *HostDB {
```

Comment:
I suppose. I wanted to avoid confusion between the hostdb wallet and the modules wallet. But I could go either way.